symbols/arm9.yml

```diff
@@ -3688,16 +3695,25 @@ arm9:
     - name: PlayBgmById
       address:
         EU: 0x2017E90
-        NA: 0x2018024
+        NA: 0x2017DF4
       description: |-
         Initializes some values and then calls SendAudioCommand to play a BGM track.
         
         Checks for DEBUG_FLAG_BGM_OFF. The volume is set to either 0 or 255 depending on the flag before calling SendAudioCommand.
         
         r0: Music ID
+    - name: StopBgmCommand
+      address:
+        EU: 0x2017F84
+        NA: 0x2017EE8
+        JP: 0x2017F40
+      description: |-
+        Stops the BGM that is being currently played by calling SendAudioCommand.
+        
+        No params.
     - name: PlayBgmByIdVolume
       address:
-        EU: 0x2017F0C
+        EU: 0x201813C
```

Comment:
Can we pull the corrections into a separate PR for auditing?

This symbol was added by @End45 from an info dump, and if the sync thinks EU/NA don't agree, I would suspect that NA is wrong rather than EU, given that End's preferred version is EU. Does the sync script have any way to tell _which_ version is wrong if there's a conflict? If not, I think we ought to have such automated overrides behind some extra step (like a command line flag on the script) to make auditing easier.

Reply:
The sync script does not resolve conflicts. If there are conflicts in pmdsky-debug, the sync will create duplicates within the decomp that result in compiler errors, and I then resolve the conflicts manually between the two projects.

I can favor the EU version here instead. The two functions are very similar, so it is possible that the US function was misidentified as matching the EU one.

Reply:
https://github.com/UsernameFodder/pmdsky-debug/pull/215

Reply:
> The sync script does not resolve conflicts. If there are conflicts in pmdsky-debug, the sync will create duplicates within the decomp that result in compiler errors, and I then resolve the conflicts manually between the two projects.

I see, that seems fine. As long as there's a forced manual step somewhere to resolve conflicts.